src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/WebHttpDispatchOperationSelectorData.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace CoreWCF.Dispatcher
+{
+    public class WebHttpDispatchOperationSelectorData
+    {
```

Comment:
I don't think this type needs to be public, especially as it has no public members.

Reply:
This has been addressed.